lib/babe/verify_test.go

```diff
@@ -525,11 +527,11 @@ func Test_verifier_verifyAuthorshipRight(t *testing.T) {
 	h := common.MustHexToHash("0x01")
 	h1 := []common.Hash{h}
 
-	mockBlockState.EXPECT().GetAllBlocksAtDepth(gomock.Any()).Return(h1)
 	mockBlockState.EXPECT().GetHeader(h).Return(types.NewEmptyHeader(), nil)
+	mockBlockState.EXPECT().GetBlockHashesBySlot(gomock.AssignableToTypeOf(uint64(0))).Return(h1, nil)
 
-	mockBlockStateErr.EXPECT().GetAllBlocksAtDepth(gomock.Any()).Return(h1)
 	mockBlockStateErr.EXPECT().GetHeader(h).Return(nil, errors.New("get header error"))
+	mockBlockStateErr.EXPECT().GetBlockHashesBySlot(gomock.AssignableToTypeOf(uint64(0))).Return(h1, nil)
```

Comment:
please don't use `gomock.AssignableToTypeOf` or `gomock.Any` when it's very easy to expect an actual value.

```suggestion
	mockBlockStateErr.EXPECT().GetBlockHashesBySlot(uint64(1)).Return(h1, nil)
```

Reply:
@timwu20 

> when it's very easy to expect an actual value.

@qdm12 Just write your suggestion. There is no need for this condescending behaviour.